src/ripple/protocol/impl/Quality.cpp

```diff
@@ -120,4 +120,35 @@ composed_quality (Quality const& lhs, Quality const& rhs)
     return Quality ((stored_exponent << (64 - 8)) | stored_mantissa);
 }
 
+Quality
+Quality::round (int digits) const
```

Comment:
It accepts values that it can apply sensibly even if they're outside the normal range. The alternatives are for it to assert, throw, return nonsense values, or return sensible values. I chose the last option.